frontend/src/screens/wallet/Swaps.tsx

```diff
@@ -0,0 +1,499 @@
+import {
+  ChevronDownIcon,
+  ChevronUpIcon,
+  ClipboardPasteIcon,
+  XCircleIcon,
+} from "lucide-react";
+import { useEffect, useState } from "react";
+import AppHeader from "src/components/AppHeader";
+import Loading from "src/components/Loading";
+import { Button } from "src/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "src/components/ui/card";
+import { Checkbox } from "src/components/ui/checkbox";
+import { Input } from "src/components/ui/input";
+import { Label } from "src/components/ui/label";
+import { LoadingButton } from "src/components/ui/loading-button";
+import { RadioGroup, RadioGroupItem } from "src/components/ui/radio-group";
+import { useToast } from "src/components/ui/use-toast";
+import { MIN_AUTO_SWAP_AMOUNT } from "src/constants";
+import { useOnchainAddress } from "src/hooks/useOnchainAddress";
+import { useSwaps } from "src/hooks/useSwaps";
+import { cn } from "src/lib/utils";
+import { SwapsSettingsResponse } from "src/types";
+import { request } from "src/utils/request";
+
+export default function Swaps() {
+  const { data: swapsSettings } = useSwaps();
+  const [swapType, setSwapType] = useState("out");
+
+  useEffect(() => {
+    const queryParams = new URLSearchParams(location.search);
+    const swapType = queryParams.get("type");
+    if (swapType) {
+      setSwapType(swapType);
+    }
+  }, []);
+
+  if (!swapsSettings) {
+    return <Loading />;
+  }
+
+  return (
+    <>
+      <AppHeader title="Swaps" />
+      <div className="flex gap-8 lg:gap-12 w-full flex-col lg:flex-row">
+        <div className="w-full max-w-lg">
+          <div className="flex items-center text-center text-foreground font-medium rounded-lg bg-muted p-1 mb-4">
+            <div
```

Comment:
![image](https://github.com/user-attachments/assets/274a2efc-fc3e-43f0-a391-472071f7afb7)

I think it's weird that it defaults to swap out but that one is on the right. Which item is selected I think is not obvious either

Reply:
Ah, the figma uses bold for the text on the selected tab